test_huge_joint_space_jump:
  ros__parameters:

    controller_name: "joint_trajectory_controller"
    wait_sec_to_publish: 3

    goal_offset: [0.5, 0.5, 0.0, 0.0, 0.0, 0.5]

    joints:
      - joint_1
      - joint_2
      - joint_3
      - joint_4
      - joint_5
      - joint_6
